admin/src/locales/de.json

```
@@ -1,4 +1,65 @@
 {
+  "bookmark": "bookmark",
+  "creation_form": {
+    "creation_for": "Schöpfung für ",
+    "enter_text": "Text eintragen",
+    "form": "Creation form",
+    "min_characters": "Mindestens 10 Zeichen eingeben",
+    "reset": "zurücksetzen",
+    "select_month": "Monat Auswählen",
+    "select_value": "Betrag Auswählen",
+    "submit_creation": "Schöpfung einreichen",
+    "toasted": "Offene Schöpfung ({value} GDD) für {email} wurde gespeichert und liegen zur Bestätigung bereit",
+    "toasted_update": "`Offene schöpfung {value} GDD) für {email} wurde geändert, liegt zur Bestätigung bereit",
+    "update_creation": "Update Schöpfung"
+  },
+  "details": "Details",
+  "e_mail": "E-Mail",
+  "firstname": "Vorname",
+  "gradido_admin_footer": "Gradido Akademie Adminkonsole",
+  "hide_details": "Details verbergen von ",
+  "lastname": "Nachname",
+  "mail": "Mail",
+  "multiple_creation_text": "Bitte wähle ein oder mehrere Mitglieder aus für die du Schöpfen möchtest.",
+  "navbar": {
+    "logout": "Logout",
+    "multi_creation": "Mehrfachschöpfung",
+    "open_creation": "offene Schöpfungen",
+    "overview": "Übersicht",
+    "user_search": "Usersuche",
+    "wallet": "Wallet"
+  },
   "not_open_creations": "keine offene Schöpfungen",
-  "open_creations": "offene Schöpfungen"
+  "open_creation": "offene Schöpfung",
+  "open_creations": "offene Schöpfungen",
+  "overlay": {
+    "confirm": {
+      "no": "Nein, nicht speichern.",
+      "question": "Willst du diese vorgespeicherte Schöpfung wirklich vollziehen und entgültig speichern?",
+      "text": "Nach dem Speichern ist der Datensatz nicht mehr änderbar und kann auch nicht mehr gelöscht werden. Bitte überprüfe genau, dass alles stimmt.",
+      "title": "Schöpfung bestätigen!",
+      "yes": "Ja, Schöpfung bestätigen und speichern!"
+    },
+    "remove": {
+      "no": "Nein, nicht löschen.",
+      "question": "Willst du die vorgespeicherte Schöpfung wirklich löschen?",
+      "text": "Nach dem Löschen gibt es keine Möglichkeit mehr diesen Datensatz wiederherzustellen. Es wird aber der gesamte Vorgang in der Logdatei als Übersicht gespeichert.",
+      "title": "Achtung! Schöpfung löschen!",
+      "yes": "Ja, Schöpfung löschen!"
+    }
+  },
+  "remove": "remove",
```

```suggestion
  "remove": "Entfernen",
```
Oder vielleicht besser `Löschen`?